app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -146,8 +148,8 @@ function initReactions() {
 }
 
 function listenForNotificationsBellClick() {
-  setTimeout(function() {
-    document.getElementById('notifications-link').onclick = function() {
+  setTimeout(() => {
+    document.getElementById('notifications-link').onclick = () => {
```

Comment:
`addEventListener` would be nice, same for all the other event handlers attached to element you find in this file

Reply:
I'll do that.